tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -12,59 +12,8 @@ build:sigbuild_remote_cache_push --remote_cache="https://storage.googleapis.com/
 # the CACHEBUSTER to the PR number.
 build --action_env=CACHEBUSTER=501872366
 
-<<<<<<< HEAD
-# Use Python 3.X as installed in container image
-build --action_env PYTHON_BIN_PATH="/usr/bin/python3"
-build --action_env PYTHON_LIB_PATH="/usr/lib/tf_python"
-build --python_path="/usr/bin/python3"
-
-# Build TensorFlow v2
-build --define=tf_api_version=2 --action_env=TF2_BEHAVIOR=1
-
-# Target the AVX instruction set
-build --copt=-mavx --host_copt=-mavx
-
-# Disable clang extention that rejects type definitions within offsetof. 
-# This was added in clang-16 by https://reviews.llvm.org/D133574.
-# Can be removed once upb is updated, since a type definition is used within
-# offset of in the current version of ubp.
-# See https://github.com/protocolbuffers/upb/blob/9effcbcb27f0a665f9f345030188c0b291e32482/upb/upb.c#L183.
-build --copt=-Wno-gnu-offsetof-extensions
-
-# Use lld as the linker
-build --linkopt="-fuse-ld=lld"
-build --linkopt="-lm"
-
-# Store performance profiling log in the mounted artifact directory.
-# The profile can be viewed by visiting chrome://tracing in a Chrome browser.
-# See https://docs.bazel.build/versions/main/skylark/performance.html#performance-profiling
-build --profile=/tf/pkg/profile.json.gz
-
-# CUDA: Set up compilation CUDA version and paths
-build --config=cuda_clang_official
-
-# ROCM: Set up compilation ROCM version and paths
-build:rocmclang --crosstool_top=@local_config_rocm//crosstool:toolchain
-build:rocmclang --crosstool_top=@local_config_rocm//:rocm_compiler=clang
-build:rocmclang --define=using_rocm_hipcc=true
-build:rocmclang --define=tensorflow_mkldnn_contraction_kernel=0
-build:rocmclang --repo_env TF_NEED_ROCM=1
-build:cudaclang --action_env=TF_ROCM_CLANG="1"
-# Disable unused-result on rocm builds.
-build:rocmclang --copt="-Wno-error=unused-result"
-
-# Test-related settings below this point.
-test:cudaclang --test_env=LD_LIBRARY_PATH="/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64"
-test:rocmclang --test_env=HSA_TOOLS_LIB=libroctracer64.so --test_sharding_strategy=disabled
-test --build_tests_only --keep_going --test_output=errors --verbose_failures=true
-# Local test jobs has to be 4 because parallel_gpu_execute is fragile, I think
-test --test_timeout=920,2400,7200,9600 --local_test_jobs=4 --run_under=//tensorflow/tools/ci_build/gpu_build:parallel_gpu_execute
-# Give only the list of failed tests at the end of the log
-test --test_summary=short
-=======
 # Build options for GPU Linux
 build --config=release_gpu_linux
->>>>>>> upstream/master
 
```

Comment:
Upstream had a big updated the build https://github.com/tensorflow/tensorflow/commit/d66aeda6ccce56c994116cac49378eb049959a38
 
are we safe on these two bazelrc files? @jayfurmanek